lib/util/http/index.ts

```diff
@@ -85,46 +85,46 @@ export class Http {
 
   async getJson<T = unknown>(
     url: string,
-    options: HttpOptions = {}
+    options?: GetOptions
   ): Promise<HttpResponse<T>> {
-    return this.requestJson<T>(url, options);
+    return this.requestJson<T>(url, { ...options });
+  }
+
+  async headJson<T = unknown>(
```

Comment:
Any reason why this was moved in the file?

Reply:
Just to co-locate it with GET request which shares same signature